widget/class-widget.php

```diff
@@ -31,7 +31,10 @@ function __construct() {
    * @param array $args     Widget arguments.
    * @param array $instance Saved values from database.
    */
-   public function widget( $args, $instance ) {
```

Comment:
I went with items to match the mark up of the other default widgets. The items inherit WordPress styling so depending on the theme they aren't always bulleted.